src/cljs/main/broadfcui/page/workspace/summary/tab.cljs

```diff
@@ -243,8 +247,13 @@
                [buttons/SidebarButton
                 {:data-test-id "open-clone-workspace-modal-button"
                  :style :light :margin :top :color :button-primary
-                 :text "Clone..." :icon :clone
-                 :disabled? (when (empty? billing-projects) (comps/no-billing-projects-message))
+                 :text (if billing-loaded?
+                         "Clone..."
+                         (spinner {:style {:margin 0}} "Getting billing info..."))
```

Comment:
if the call to get billing fails, will this be an infinite spinner?

Reply:
Yeah, I guess it would. I could add an error state, with a `:disabled?` modal that says to reload the page.